### PR TITLE
Wrong typing of the Grant Token endpoint

### PR DESCRIPTION
**Pull request by seba-aln**
fix: Fixed wrong type annotation for grant token response value

docs: updated compatibility list

### CODE REVIEW

File: examples/GrantToken.php [hunk 1]
```diff
@@ -2,35 +2,38 @@
...
+
+    /** @var PNAccessManagerTokenResult */
+    $parsedToken = $pubnub->parseToken($token);
+
+    $tokensTTL = $parsedToken->getTtl();
+    $tokensMyChannelRead = $parsedToken->getChannelResource('my-channel')->hasRead();
+    $tokensMyChannelWrite = $parsedToken->getChannelResource('my-channel')->hasWrite();
+
+    print("Token TTL: $tokensTTL\n");
+    print("Token My Channel Read: " . (int)$tokensMyChannelRead . "\n");
```
Comment (marcin-cebo) [inline examples/GrantToken.php:34]: IMHO for readability there could be space between (int) and $tokensMyChannelRead

**Review by marcin-cebo — APPROVED** [review]

**Comment by seba-aln** [comment]
@pubnub-release-bot release

**Comment by seba-aln** [comment]
@pubnub-release-bot release

File: examples/GrantToken.php (hunk 1 above)
Comment (seba-aln) [reply examples/GrantToken.php:34]: It may or may not have space. For me the typecast is part of expression so better when it's joined

**Review by kazydek — APPROVED** [review]

**Pull request merged**